nexus/db-model/src/inventory.rs

```diff
@@ -532,6 +661,13 @@ pub struct InvRootOfTrust {
     pub slot_boot_pref_persistent_pending: Option<HwRotSlot>,
     pub slot_a_sha3_256: Option<String>,
     pub slot_b_sha3_256: Option<String>,
+    pub stage0_fwid: Option<String>,
+    pub stage0next_fwid: Option<String>,
+
+    pub slot_a_status: Option<RotImageError>,
+    pub slot_b_status: Option<RotImageError>,
+    pub stage0_status: Option<RotImageError>,
+    pub stage0next_status: Option<RotImageError>,
```

Comment:
Small nit - I'm not sure about naming these fields `*_status` when they can only contain errors. Is a value of `None` good? Would this be clearer either as `*_error` or by keeping `*_status` but changing the types to `Result<(), RotImageError>`?

Reply:
The actual gateway message does have this as `Result<(), ImageError>` but iirc I ran into some weirdness with trying to get that into the database so I made it an Option instead. I can change the name to make it clear it's only an error.